server/public/styles.css

```diff
@@ -20,3 +19,22 @@ body {
 .width-150 {
   width: 150px;
 }
+
+.fa-grin {
+  font-size: 5rem;
+}
+
+.user-icon-container {
+  width: 6rem;
+  height: 6rem;
+  background-color: white;
+  border-radius: 50%;
+}
+
+.review-content {
+  font-size: 1.2rem;
+}
+
+.fa-star {
+  color: #ff8e00;
+}
```

Comment:
Avoid styling CSS classes from libraries, instead add your own additional class like `.yellow` and use it with the library class.